logbook-servlet/src/main/java/org/zalando/logbook/servlet/LogbookFilter.java

```diff
@@ -17,9 +19,11 @@
 import java.util.UUID;
 
 import static javax.servlet.DispatcherType.ASYNC;
+import static lombok.AccessLevel.PRIVATE;
 import static org.apiguardian.api.API.Status.STABLE;
 
 @API(status = STABLE)
+@AllArgsConstructor(access = PRIVATE)
```

Comment:
Intentionally hiding the constructor? Is there no use case for anyone programatically instantiating a `LogbookFilter` with a explicit `FormRequestMode`?

Reply:
I prefer to expose only the constructor with required arguments. So that a user get's the default by invoking the constructor but can subsequently override/configure things as needed using `with*` methods (convention over configuration).

Reply:
Ah I've missed the `@With` annotation on the field, makes sense :+1: